lib/cms/models/cms_models.dart

```diff
@@ -44,3 +111,27 @@ class CarouselDataItem {
     return 'CarouselDataItem{id: $id, link: $link, media: $media}';
   }
 }
+
+class SpaceValueConverter extends JsonConverter<double, String> {
+  const SpaceValueConverter();
+
+  @override
+  double fromJson(String json) {
+    // Impossible to do better with static consts (as opposed to enum)
+    switch (json) {
+      case 'sm':
+        return FlexSizes.sm;
+      case 'md':
+        return FlexSizes.md;
+      case 'lg':
+        return FlexSizes.lg;
+      default:
+        throw Exception('Unknown value for space $json');
```

Comment:
Can we default this to `FlexSizes.spacerItems`?

Reply:
Oh definitely yes 👍

Reply:
Fixes in next PR